ynv/ynvimport.py

```diff
@@ -52,19 +52,26 @@ def portals_to_obj(portals):
     return pobj
 
 
+material_cache = {}
+
 def get_material(flags):
```

Comment:
Pass `material_cache` as a parameter to `get_material`, and define it as a local variable of `polygons_to_obj` instead of a global. A global will cause issues, like multiple navmeshes using materials of other navmeshes or keeping references to deleted materials (which will crash).